tests/integration/components/o-s-s/social-media-badge-test.ts

```diff
@@ -0,0 +1,113 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import setupOnerror from '@ember/test-helpers/setup-onerror';
```

Comment:
`setupOnerror` can be imported in line 3 also, btw :)

Reply:
Fix imports in ad4f3c0940b3fe3b00c6f3a59e1a2d4ebe7663e7